server/backend-shared/src/main/scala/cool/graph/shared/errors/Errors.scala

```diff
@@ -20,7 +20,7 @@ object CommonErrors {
   case class QueriesNotAllowedForProject(projectId: String) extends UserFacingError(s"Queries are not allowed for the project with id '$projectId'", 1002)
 
   case class MutationsNotAllowedForProject(projectId: String)
-      extends UserFacingError(s"The project '$projectId' is currently in read-only mode. Please try again in a few minutes", 1003)
+    extends UserFacingError(s"The project '$projectId' is currently in read-only mode. Please try again in a few minutes", 1003)
```

Comment:
Are you sure your scalafmt is set up correctly?

Reply:
It wasn't 👍